stac_fastapi/pgstac/tests/clients/test_postgres.py

```diff
@@ -0,0 +1,158 @@
+    resp = await app_client.post(
+        "/collections",
+        json=in_json,
+    )
+    assert resp.status_code == 200
+    post_coll = Collection.parse_obj(resp.json())
+    assert in_coll.dict(exclude={"links"}) == post_coll.dict(exclude={"links"})
+    resp = await app_client.get(f"/collections/{post_coll.id}")
+    assert resp.status_code == 200
+    get_coll = Collection.parse_obj(resp.json())
+    assert post_coll.dict(exclude={"links"}) == get_coll.dict(exclude={"links"})
+
+
+@pytest.mark.asyncio
+async def test_update_collection(app_client, load_test_data, load_test_collection):
+    in_coll = load_test_collection
+    in_coll.keywords.append("newkeyword")
+
+    resp = await app_client.put("/collections", json=in_coll.dict())
+    assert resp.status_code == 200
+
+    resp = await app_client.get(f"/collections/{in_coll.id}")
+    assert resp.status_code == 200
+
+    get_coll = Collection.parse_obj(resp.json())
+    assert in_coll.dict(exclude={"links"}) == get_coll.dict(exclude={"links"})
+    assert "newkeyword" in get_coll.keywords
+
+
+@pytest.mark.asyncio
+async def test_delete_collection(
+    app_client, load_test_data: Callable, load_test_collection
+):
+    in_coll = load_test_collection
+
+    resp = await app_client.delete(f"/collections/{in_coll.id}")
+    assert resp.status_code == 200
+
+    resp = await app_client.get(f"/collections/{in_coll.id}")
+    assert resp.status_code == 404
+
+
+@pytest.mark.asyncio
+async def test_create_item(app_client, load_test_data: Callable, load_test_collection):
+    coll = load_test_collection
+
+    in_json = load_test_data("test_item.json")
+    in_item = Item.parse_obj(in_json)
+    resp = await app_client.post(
+        "/collections/{coll.id}/items",
+        json=in_json,
+    )
+    assert resp.status_code == 200
+
+    post_item = Item.parse_obj(resp.json())
+    assert in_item.dict(exclude={"links"}) == post_item.dict(exclude={"links"})
+
+    resp = await app_client.get(f"/collections/{coll.id}/items/{post_item.id}")
+
+    assert resp.status_code == 200
+    get_item = Item.parse_obj(resp.json())
+    assert in_item.dict(exclude={"links"}) == get_item.dict(exclude={"links"})
+
+
+@pytest.mark.asyncio
+async def test_update_item(
+    app_client, load_test_data: Callable, load_test_collection, load_test_item
+):
+    coll = load_test_collection
+    item = load_test_item
+
+    item.properties.description = "Update Test"
+
+    resp = await app_client.put(f"/collections/{coll.id}/items", json=item.dict())
+    assert resp.status_code == 200
+
+    resp = await app_client.get(f"/collections/{coll.id}/items/{item.id}")
+    assert resp.status_code == 200
+
+    get_item = Item.parse_obj(resp.json())
+    assert item.dict(exclude={"links"}) == get_item.dict(exclude={"links"})
+    assert get_item.properties.description == "Update Test"
+
+
+@pytest.mark.asyncio
+async def test_delete_item(
+    app_client, load_test_data: Callable, load_test_collection, load_test_item
```

Comment:
```suggestion
    app_client, load_test_collection, load_test_item
```

There's a lot of tests where this fixture isn't consumed.  Looks like you have `load_test_data` and `load_test_collection`/`load_test_item` which themselves are using `load_test_data`.  Please be consistent with this pattern, either use `load_test_data` everywhere or `load_test_*`.